decidim-proposals/config/locales/ca.yml

```diff
@@ -453,7 +453,7 @@ ca:
               title: La següent acció és irreversible
             success: L'esborrany col·laboratiu s'ha retirat amb èxit.
         compare:
-          mine_is_different: El meu esborrany col·laboratiu és diferent
```

Comment:
Please only change texts at `en.yml` since it's the source of truth for Crowdin

Reply:
Thanks